examples/gatsby/src/components/ProductGallery/ProductGallery.js

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
```

Comment:
Similar to above -- how come we're importing `*`?

Reply:
The `import * as React from 'react` pattern ([is used in TypeScript React projects](https://stackoverflow.com/a/55285991/6387812)). Since this isn't a TypeScript project, I'm happy to change it to `import React from 'react'` if it helps with consistency between example projects.

Reply:
Yeah, none of our current React TS projects import in that way. Explicit imports > implicit exports IMO. It's also just a good pattern / practice to promote b/c it's required for tree-shaking to work (although pretty much all of React gets bundled anyway, that's not the case for other libraries).